### ISSUE TITLE
Status bars for LMS

### ISSUE THREAD

**Issue by ColemanDunn** [runelite/runelite#11378]
Runelite's status bars max out where your current player's max hp is.  When you start an LMS game your combat stats including HP go to 99 and def goes to 75. 

So if I make a new account with 10 hp, my hp level is boosted to 99 but my max hp stays at 10 and in turn, my health status bar also stays at 10. This gets very confusing when fighting as you don't know what proportion of health you are missing when glancing at the status bar. 

The number on the status bar is correct but it would be nice if the graphic corresponding to your current hp in proportion form 0-99. 

So in other words, if you are in LMS your status bar total health in the health bar code should be set to 99 in order to properly display your correct proportion of hp left. This would not apply to prayer as prayer does not drain in LMS.

**Comment by maxbbender** [comment]
Very excited for this! Was about to report an issue but glad it's already being worked on :) Thanks @pwatts6060 !

**Comment by maxbbender** [comment]
Think this will be merged in time for Group Ironman release? Big QOL for grinding those rune pouches 😄

**Comment by Nightfirecat** [comment]
Added in c7ea7a7e145467b0c50a619e10fbcbbcd105f0db
